diff --git trilinos/CMakeLists.txt trilinos/CMakeLists.txt
index 7f789df..10dd944 100644
--- trilinos/CMakeLists.txt
+++ trilinos/CMakeLists.txt
@@ -59,7 +59,7 @@
 
 # To be safe, define your minimum CMake version.  This may be newer than the
 # min required by TriBITS.
-CMAKE_MINIMUM_REQUIRED(VERSION 3.10.0 FATAL_ERROR)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1 FATAL_ERROR)
 
 # Must set the project name as a variable at very beginning before including anything else
 # We set the project name in a separate file so CTest scripts can use it.
diff --git trilinos/cmake/tribits/core/package_arch/TribitsConstants.cmake trilinos/cmake/tribits/core/package_arch/TribitsConstants.cmake
index bcf1975..fd50080 100644
--- trilinos/cmake/tribits/core/package_arch/TribitsConstants.cmake
+++ trilinos/cmake/tribits/core/package_arch/TribitsConstants.cmake
@@ -39,7 +39,7 @@
 
 # Define the TriBITS minimum required CMake version
 
-SET(TRIBITS_CMAKE_MINIMUM_REQUIRED 3.10.0)
+SET(TRIBITS_CMAKE_MINIMUM_REQUIRED 3.1)
 
 MACRO(TRIBITS_ASESRT_MINIMUM_CMAKE_VERSION)
 
diff --git trilinos/packages/teuchos/CMakeLists.txt trilinos/packages/teuchos/CMakeLists.txt
index 3a6a080..d0e7b43 100644
--- trilinos/packages/teuchos/CMakeLists.txt
+++ trilinos/packages/teuchos/CMakeLists.txt
@@ -416,7 +416,7 @@ ELSE(TRIBITS_PROCESSING_PACKAGE)
 
 # This CMakeLists.txt file is being processed as the TriBITS projects's base
 # CMakeLists.txt file!  (See comments at bottom of this file.)
-CMAKE_MINIMUM_REQUIRED(VERSION 3.10.0 FATAL_ERROR)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1 FATAL_ERROR)
 INCLUDE("${CMAKE_CURRENT_SOURCE_DIR}/ProjectName.cmake")
 PROJECT(${PROJECT_NAME} NONE)
 SET(${PROJECT_NAME}_TRIBITS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/tribits"
